### ISSUE TITLE
Assign rights to ETD

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/alexandria-v2#268]
Extend the existing ETD Importer to parse the rights in the XML file contained in the proquest zip file

**GIVENS**
* There will be an existing ETD object that has a system ID, title, creator, and primary filename
* Downloads will be in a server directory "#{downloads_root}/etd/proquest"
* Supplemental files will be embedded in a zip file with a name in the format:  etdadmin_upload_331128.zip
* Valid files for import must contain
    * at least one PDF matching the primary file name 
    * at least one XML file matching the primary file name
* Importer runs from the command line in the foreground for the time being

**Comment by val99erie** [comment]
This story probably depends on #266

**Comment by mark-dce** [comment]
Superseded by #244
